### PR TITLE
Add sprite font glyphs for Æ and Ø

### PR DESCRIPTION
**Pull request by Gymnasiast**
The glyphs are fine, but the code to detect the width does not work properly, and I have a feeling some things could be done better. I am opening this pull request to get advice/help with that.

Requires #7801 to be merged first.

### CODE REVIEW

File: src/openrct2/drawing/ImageImporter.cpp [hunk 1]
```diff
@@ -362,15 +362,15 @@ const PaletteBGRA ImageImporter::StandardPalette[256] =
...
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
-    { 0, 0, 0, 255 },
+    { 0, 0, 1, 255 },
```
Comment (AaronVanGeffen) [inline src/openrct2/drawing/ImageImporter.cpp]: Why did you change these? The third octet represents red, not an index, right?

File: src/openrct2/sprites.h [hunk 2]
```diff
@@ -840,6 +840,14 @@ enum {
     
     SPR_G2_EYEDROPPER = SPR_G2_BEGIN + 123,
 
+    SPR_G2_CHAR_BEGIN = SPR_G2_BEGIN + 124,
```
Comment (AaronVanGeffen) [inline src/openrct2/sprites.h]: I think we ought to make constants for each of the different fonts (small, normal, tiny).

File: src/openrct2/sprites.h [hunk 3]
```diff
@@ -840,6 +840,14 @@ enum {
...
+    SPR_G2_AE_UPPER = SPR_G2_CHAR_BEGIN,
+    SPR_G2_AE_LOWER = SPR_G2_CHAR_BEGIN + 1,
+    SPR_G2_O_STROKE_UPPER = SPR_G2_CHAR_BEGIN + 2,
+    SPR_G2_O_STROKE_LOWER = SPR_G2_CHAR_BEGIN + 3,
+
+    SPR_G2_GLYPH_COUNT = (SPR_G2_O_STROKE_LOWER - SPR_G2_CHAR_BEGIN) + 1,
```
Comment (AaronVanGeffen) [inline src/openrct2/sprites.h]: This is off, I think, because you have only taken the small font into account.

Comment (Gymnasiast) [reply src/openrct2/sprites.h]: This is how the vanilla glyph count works as well. It's more useful like this anyway.

File: src/openrct2/drawing/ImageImporter.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2/drawing/ImageImporter.cpp]: Mostly to help distinguish the separate indices a bit. I can revert this bit if preferred.

Comment (IntelOrca) [reply src/openrct2/drawing/ImageImporter.cpp]: Why didn't you just use comments?

Comment (Gymnasiast) [reply src/openrct2/drawing/ImageImporter.cpp]: I wanted it to help distinguish colours in image modifying programs. Comments don't help there.

Again, I can revert this bit.

Comment (IntelOrca) [reply src/openrct2/drawing/ImageImporter.cpp]: Oh I see. In that case shouldn't you do increments of 20 or perhaps different hues?
